.travis.yml

```diff
@@ -27,6 +27,33 @@ jobs:
         - pip install elasticsearch sh boto3
       script: ./server/bin/unittests
 
+    - stage: "Agent Pytests"
```

Comment:
Be sure to review https://github.com/distributed-system-analysis/pbench/pull/1583 which is going to either make this easier for you to change, or harder because of merge conflicts. =)

Reply:
That patch is using tox to run the already existing unit test scripts but I am using pytest framework instead. I can modify this definitions to use tox to run my pytests.